presidio-image-redactor/tests/integration/test_dicom_image_redactor_engine_integration.py

```diff
@@ -33,33 +57,33 @@ def mock_engine():
         (Path(RESOURCES_DIR2, "2_ORIGINAL.dcm")),
     ],
 )
-def test_redact_image_correctly(
-    mock_engine: DicomImageRedactorEngine, dcm_filepath: Path
-):
-    """Test the redact function
+@pytest.mark.parametrize("engine_builder", all_engines_required())
+def test_redact_image_correctly(engine_builder, dcm_filepath: Path):
+    """Test the redact function.
 
     Args:
-        mock_engine (DicomImageRedactorEngine): Mock instance.
+        engine (DicomImageRedactorEngine): Mock instance.
```

Comment:
**nit**: Typo "engine" -> "engine_builder"

Reply:
I recommend doing a ctrl + F since this occurs in multiple docstrings in this file.